application/views/admin/survey/subview/accordion/_generaloptions_panel.php

```diff
@@ -319,7 +319,7 @@ function checkSelect2Languages(mylangs) {
         <div class="mb-3" >
             <label class=" form-label" for='template'><?php eT("Theme:"); ?></label>
             <div class="">
-                <select id='template' class="form-select" name='template' data-updateurl='<?php echo App()->createUrl('themeOptions/getPreviewTag') ?>'
+                <select id='template' style="width:100%;" class="form-select activate-search" name='template' data-updateurl='<?php echo App()->createUrl('themeOptions/getPreviewTag') ?>'
```

Comment:
style is needed ?

Reply:
Yes, as to no have the dropdown trimmed

Reply:
There are no form-XX class for this … :(